extensions/adobe/experience/edge-autofilled-environment-details.schema.json

```diff
@@ -43,15 +43,53 @@
         }
       }
     },
-    "edge-environment-geo": {
+    "edge-environment-countrycode": {
       "properties": {
         "xdm:placeContext": {
           "type": "object",
           "properties": {
             "xdm:geo": {
               "title": "Geo",
-              "$ref": "https://ns.adobe.com/xdm/common/geo",
-              "description": "The geographic location where the experience was delivered based on the IP address from the client request."
+              "$ref": "https://ns.adobe.com/xdm/common/geo#/definitions/countrycode"
+            }
+          }
+        }
+      }
+    },
+    "edge-environment-postalcode": {
+      "properties": {
+        "xdm:placeContext": {
+          "type": "object",
+          "properties": {
+            "xdm:geo": {
+              "title": "Geo",
+              "$ref": "https://ns.adobe.com/xdm/common/geo#/definitions/postalcode"
+            }
+          }
+        }
+      }
+    },
+    "edge-environment-latitude": {
+      "properties": {
+        "xdm:placeContext": {
+          "type": "object",
+          "properties": {
+            "xdm:geo": {
+              "title": "Geo",
+              "$ref": "http://schema.org/GeoCoordinates#/definitions/latitude"
```

Comment:
I'm not sure if this is the correct way to reference a field for a data type which is extended by another data type. In this case, Geo extends GeoCoordinates and the path for this field in this field group should be `placeContext/geo/_schema/latitude`. Does this definition look correct, or do I need another nested property for the `schema` namespace?